src/dev/ci_setup/setup.sh

```diff
@@ -180,3 +180,23 @@ if [ "$GIT_CHANGES" ]; then
   echo -e "$GIT_CHANGES\n"
   exit 1
 fi
+
+###
+### github-checks-reporter kill switch. Remove to disable
+###
+export CHECKS_REPORTER_ACTIVE=true
+
+###
+### Implements github-checks-reporter kill switch when scripts are called from the command line
+### $@ - all arguments
+###
+function checks-reporter-with-killswitch() {
+  if [ "$CHECKS_REPORTER_ACTIVE" = true ] && [[ "$JOB_NAME" = "elastic+kibana+pull-request"* ]] ; then
```

Comment:
Now checking job name

Reply:
No need, since we only set `CHECKS_REPORTER_ACTIVE=true` when the name matches